fatoptimizer/methods.py

```diff
@@ -35,6 +41,44 @@ def add(obj_type, name, *args, **kw):
     add(bytes, 'decode', (0, 2), str, str,
         check_args=check_encoding,
         exceptions=UnicodeDecodeError)
+    add(bytes, 'endswith', (1, 3), tuple, int, int)
+    add(bytes, 'find', (1, 3), bytes, int, int)     # FIXME: Here "sub"(that is to be found) can be both bytes and int
```

Comment:
You can use the "object" type to accept any type, and then use check_args to check manually arguments types and values.